issue_02/script.js

```diff
@@ -0,0 +1,11 @@
+const ul = document.getElementById('js-ul');
+const a = document.createElement('a');
+a.href = '1.html';
+a.textContent = 'これです';
+const img = document.createElement('img');
+img.src = 'bookmark.png';
+img.alt = 'ブックマーク';
+a.appendChild(img);
+const li = document.createElement('li');
+li.appendChild(a);
+ul.appendChild(li);
```

Comment:
IMO
It can be written in abbreviated form like this.
Do some research on the Mezzot chain.
```suggestion
ul.appendChild(li).appendChild(a);
```

Reply:
I rewrote the code to the one you suggested.
https://github.com/yoshikawa-akr/js-lessons/pull/2/commits/297e1347b7f62e22363f973dc81f02e202795f3e

Thanks to your review here, I was able to learn about method chaining. Thank you very much!